devlib/module/cpufreq.py

```diff
@@ -152,10 +152,14 @@ def set_governor_tunables(self, cpu, governor=None, **kwargs):
         valid_tunables = self.list_governor_tunables(cpu)
         for tunable, value in kwargs.iteritems():
             if tunable in valid_tunables:
+                path = '/sys/devices/system/cpu/{}/cpufreq/{}/{}'.format(cpu, governor, tunable)
                 try:
-                    path = '/sys/devices/system/cpu/{}/cpufreq/{}/{}'.format(cpu, governor, tunable)
                     self.target.write_value(path, value)
-                except TargetError:  # May be an older kernel
+                except TargetError:
+                    if te.target.file_exists(path):
```

Comment:
Eh yeah. Thought I tested this (I saw the error I wanted to see) so may have misunderstood the whole thing, will have to rewrite.